packages/components/src/GridLayout/Tile/Body/TileBody.component.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render } from '@testing-library/react';
+import TileBody from './TileBody.component';
+
+describe("tile's body", () => {
+	it('should compute tile body', () => {
+		// when
+		const { container } = render(<TileBody>my body</TileBody>);
+
+		// then
+		expect(container.firstChild).toMatchSnapshot();
```

Comment:
You still use snapshots 🤔 
Why not as this snapshot is quite small, but you can also check if `my body` is visible.

Reply:
Yes I am ok to keep a snapshot on `it('should render'` test as this is what it does.
It helps me to detect change after lib upgrade or component HTML change.